.tool-versions

```diff
@@ -1,2 +1,3 @@
-ruby 2.6.9
+ruby 3.0.4
 nodejs 16.13.1
+bundle 2.2.33
```

Comment:
Fyi asdf isn't managing bundler so this won't do anything.

Reply:
You probably want https://github.com/asdf-vm/asdf-ruby#default-gems in the base image

Reply:
```suggestion
```